README.md

```diff
@@ -48,6 +48,8 @@ If the basic registers change you can restart your register so that it sees the
 - Install and run postgres db
 - Checkout project
 - Run the `./go` script to set everything up
+- `$ git submodule init`
+- `$ git submodule update`
```

Comment:
After further research we can replace `git submodule init` with `git submodule update --init`, this will pull down the submodules based on the commit version within the project and initialise if they aren't already. We need to add further documentation to actually update the submodules which is `git submodule foreach git pull origin master`